eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/EventHandler.java

```diff
@@ -72,13 +71,10 @@ public class EventHandler extends AbstractHttpHandler {
      * Constructs a new instance with the specified connector plugin type and HTTP handler manager.
      *
      * @param connectorPluginType The name of event storage connector plugin.
-     * @param httpHandlerManager httpHandlerManager Manages the registration of {@linkplain com.sun.net.httpserver.HttpHandler HttpHandler}
-     *                           for an {@link com.sun.net.httpserver.HttpServer HttpServer}.
      */
     public EventHandler(
-        String connectorPluginType,
-        HttpHandlerManager httpHandlerManager) {
-        super(httpHandlerManager);
+        String connectorPluginType) {
```

Comment:
this param seems like `storagePluginType`, as well as the comment on this Class, not the `connector plugin type`.

Reply:
This will be optimized in the future.Thank you.